### PR TITLE
CEDS-1490 Remove CustomExportsBaseSpec

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: test/connectors/CustomsDeclareExportsConnectorSpec.scala [hunk 1]
```diff
@@ -16,23 +16,40 @@
...
-class CustomsDeclareExportsConnectorSpec extends CustomExportsBaseSpec {
+import scala.concurrent.ExecutionContext.global
+
+class CustomsDeclareExportsConnectorSpec extends UnitSpec with ScalaFutures {
   import CustomsDeclareExportsConnectorSpec._
 
+  val injector = GuiceApplicationBuilder().injector()
+  val appConfig = injector.instanceOf[AppConfig]
+  val hc: HeaderCarrier =
+    HeaderCarrier(
```
Comment (ed0906) [inline test/connectors/CustomsDeclareExportsConnectorSpec.scala]: Pretty sure you can just do HeaderCarrier() and there is sensible defaults

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorSpec.scala]: I'll check it, if it's possible I'll remove this from all places, thanks :)

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorSpec.scala]: Authorisation is necessary, DateTime not, I'll change it

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorSpec.scala]: Changed

File: test/services/NrsServiceSpec.scala [hunk 2]
```diff
@@ -16,22 +16,37 @@
...
+  val signedInUser = newUser("12345", "external1")
 
   "NrsService " should {
 
     "submit user submission to NRS service with a valid submissionId" in {
 
       submitNrsRequest()
 
-      val nrsResponse = nrsService.submit("conversationId1", "payload 1", "ducr1")
+      val nrsResponse = nrsService.submit("conversationId1", "payload 1", "ducr1")(hc, global, signedInUser)
```
Comment (MaciejRewera) [inline test/services/NrsServiceSpec.scala]: Why not using implicits anymore?

File: test/services/audit/AuditServiceSpec.scala [hunk 3]
```diff
@@ -116,21 +142,24 @@ trait AuditTestSupport extends CustomExportsBaseSpec {
...
 
-  def mockSendCompletePayload(result: AuditResult = Success) =
+  def mockSendCompletePayload(result: AuditResult = Success): OngoingStubbing[Future[AuditResult]] =
     when(
       mockAuditConnector.sendExtendedEvent(ArgumentMatchers.refEq(extendedEvent, "eventId", "generatedAt"))(
         ArgumentMatchers.any[HeaderCarrier],
         ArgumentMatchers.any[ExecutionContext]
       )
-    ) thenReturn Future.successful(result)
+    ).thenReturn(Future.successful(result))
```
Comment (MaciejRewera) [inline test/services/audit/AuditServiceSpec.scala]: 👍

File: test/services/audit/AuditServiceSpec.scala [hunk 4]
```diff
@@ -116,21 +142,24 @@ trait AuditTestSupport extends CustomExportsBaseSpec {
 
   val auditFailure = Failure("Event sending failed")
 
-  val auditService = new AuditService(mockAuditConnector, appConfig)
+  val auditService = new AuditService(mockAuditConnector, appConfig)(global)
 
-  def mockSendEvent(evenToAudit: DataEvent = event, result: AuditResult = Success) =
+  def mockSendEvent(
+    evenToAudit: DataEvent = event,
```
Comment (MaciejRewera) [inline test/services/audit/AuditServiceSpec.scala]: There is a typo. I think you meant `eventToAudit`

File: test/services/NrsServiceSpec.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply test/services/NrsServiceSpec.scala]: It's just test, with explicit parameters you can see what you need. I don't see the point to using implicits like this in tests, but I can come back to implicit if you want.

File: test/services/audit/AuditServiceSpec.scala (hunk 4 above)
Comment (PatrykRudnicki) [reply test/services/audit/AuditServiceSpec.scala]: not mine, fixed

**Pull request merged**